dist/images/ovnkube.sh

```diff
@@ -1030,6 +1034,258 @@ ovn-master() {
+ovn-cluster-manager() {
+  trap 'kill $(jobs -p); exit 0' TERM
+  check_ovn_daemonset_version "3"
+
+  echo "=============== ovn-cluster-manager (wait for ready_to_start_node) ========== MASTER ONLY"
+  wait_for_event ready_to_start_node
+
+  hybrid_overlay_flags=
+  if [[ ${ovn_hybrid_overlay_enable} == "true" ]]; then
+    hybrid_overlay_flags="--enable-hybrid-overlay"
+    if [[ -n "${ovn_hybrid_overlay_net_cidr}" ]]; then
+      hybrid_overlay_flags="${hybrid_overlay_flags} --hybrid-overlay-cluster-subnets=${ovn_hybrid_overlay_net_cidr}"
+    fi
+  fi
+  echo "hybrid_overlay_flags: ${hybrid_overlay_flags}"
+
+  ovn_v4_join_subnet_opt=
+  if [[ -n ${ovn_v4_join_subnet} ]]; then
+      ovn_v4_join_subnet_opt="--gateway-v4-join-subnet=${ovn_v4_join_subnet}"
+  fi
+  echo "ovn_v4_join_subnet_opt: ${ovn_v4_join_subnet_opt}"
+
+  ovn_v6_join_subnet_opt=
+  if [[ -n ${ovn_v6_join_subnet} ]]; then
+      ovn_v6_join_subnet_opt="--gateway-v6-join-subnet=${ovn_v6_join_subnet}"
+  fi
+  echo "ovn_v6_join_subnet_opt: ${ovn_v6_join_subnet_opt}"
+
+  multicast_enabled_flag=
+  if [[ ${ovn_multicast_enable} == "true" ]]; then
+      multicast_enabled_flag="--enable-multicast"
+  fi
+  echo "multicast_enabled_flag: ${multicast_enabled_flag}"
+
+  multi_network_enabled_flag=
+  if [[ ${ovn_multi_network_enable} == "true" ]]; then
+	  multi_network_enabled_flag="--enable-multi-network"
+  fi
+  echo "multi_network_enabled_flag: ${multi_network_enabled_flag}"
+
+  ovnkube_cluster_manager_metrics_bind_address="${metrics_endpoint_ip}:9411"
+  echo "ovnkube_cluster_manager_metrics_bind_address: ${ovnkube_cluster_manager_metrics_bind_address}"
+
+  local ovnkube_metrics_tls_opts=""
+  if [[ ${OVNKUBE_METRICS_PK} != "" && ${OVNKUBE_METRICS_CERT} != "" ]]; then
+    ovnkube_metrics_tls_opts="
+        --node-server-privkey ${OVNKUBE_METRICS_PK}
+        --node-server-cert ${OVNKUBE_METRICS_CERT}
+      "
+  fi
+  echo "ovnkube_metrics_tls_opts: ${ovnkube_metrics_tls_opts}"
+
+  ovnkube_config_duration_enable_flag=
+  if [[ ${ovnkube_config_duration_enable} == "true" ]]; then
+    ovnkube_config_duration_enable_flag="--metrics-enable-config-duration"
+  fi
+  echo "ovnkube_config_duration_enable_flag: ${ovnkube_config_duration_enable_flag}"
+
+  echo "=============== ovn-cluster-manager ========== MASTER ONLY"
+  /usr/bin/ovnkube \
+    --init-cluster-manager ${K8S_NODE} \
+    --cluster-subnets ${net_cidr} --k8s-service-cidr=${svc_cidr} \
+    --loglevel=${ovnkube_loglevel} \
+    --logfile-maxsize=${ovnkube_logfile_maxsize} \
+    --logfile-maxbackups=${ovnkube_logfile_maxbackups} \
+    --logfile-maxage=${ovnkube_logfile_maxage} \
+    ${hybrid_overlay_flags} \
+    ${ovn_v4_join_subnet_opt} \
+    ${ovn_v6_join_subnet_opt} \
+    --pidfile ${OVN_RUNDIR}/ovnkube-cluster-manager.pid \
+    --logfile /var/log/ovn-kubernetes/ovnkube-cluster-manager.log \
+    ${ovnkube_metrics_tls_opts} \
+    ${multicast_enabled_flag} \
+    ${ovnkube_config_duration_enable_flag} \
```

Comment:
remove - its only needed in master mode or network controller mgr mode.

Reply:
Ack